### PR TITLE
Feature/#88 agency partner detail overview

### PR DESCRIPTION
**Pull request by kenzo23312**
Agency partner detail overview

### CODE REVIEW

File: frontend/src/components/common/headerNavigation.js [hunk 1]
```diff
@@ -65,15 +66,16 @@ class HeaderNavigation extends Component {
...
                 <IconButton onClick={handleBackButton}>
                   <KeyboardArrowLeft />
                 </IconButton>
-                : null
-              }
-              <Typography type="headline">
-                {title}
-              </Typography>
+                : null }
+              { title ?
```
Comment (marcindo) [inline frontend/src/components/common/headerNavigation.js]: I think it's better to detect `typeof title === 'string'`

File: frontend/src/components/common/cell/itemRowCellDivider.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <div>
+      <Grid container align="center" justify="space-between" direction="row">
+        <Grid className={classes.row} item xs={8}>
+          <Typography color={labelSecondary ? 'secondary' : 'inherit'} type="body1" >{label}</Typography>
+        </Grid>
+        <Grid item xs={4} >
+          <Typography type="body1" >{content}</Typography>
+        </Grid>
+      </Grid>
+      {!divider ?
```
Comment (marcindo) [inline frontend/src/components/common/cell/itemRowCellDivider.js]: just for consistency, ? and : should be on new line followed by elements

File: frontend/src/components/common/headerNavigation.js [hunk 3]
```diff
@@ -65,15 +66,16 @@ class HeaderNavigation extends Component {
         <Grid align="center" className={classes.container} container>
           <Grid item xs={8}>
             <div className={tabs ? classes.alignItems : classes.alignItemsPadding}>
-              {backButton ?
+              { backButton ?
```
Comment (marcindo) [inline frontend/src/components/common/headerNavigation.js]: move ? to next line please

File: frontend/src/components/common/list/headerList.js [hunk 4]
```diff
@@ -28,12 +28,12 @@ class HeaderList extends Component {
...
 
   render() {
-    const { classes, header } = this.props;
+    const { classes, header, headerObject } = this.props;
     return (
       <Paper>
         <Grid direction="column" className={classes.container} container gutter={0}>
           <Grid className={classes.header} item>
-            {createElement(header)}
+            {header ? createElement(header) : headerObject}
```
Comment (marcindo) [inline frontend/src/components/common/list/headerList.js]: header is rather pointless as you can just detect if headerObject was provided

File: frontend/src/components/organizationProfile/hq/hqProfileHeader.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
```
Comment (marcindo) [inline frontend/src/components/organizationProfile/hq/hqProfileHeader.js]: is this header different for country profiles? If not maybe it shouldn't have hq added to the name?

File: frontend/src/components/partners/partnerFilter.js [hunk 6]
```diff
@@ -14,12 +14,46 @@ const messages = {
   labels: {
     search: 'Search',
     country: 'Country',
+    verificationStatus: 'Verification Status',
+    typeOfOrganization: 'Type of Organization',
+    sectorArea: 'Sector & Area of specialization',
+    populations: 'Populations of concer',
```
Comment (marcindo) [inline frontend/src/components/partners/partnerFilter.js]: concern

File: frontend/src/components/partners/partnerFilter.js [hunk 7]
```diff
@@ -14,12 +14,46 @@ const messages = {
...
   },
   clear: 'clear',
 };
 
+const VERIFICATION_MENU = [
```
Comment (marcindo) [inline frontend/src/components/partners/partnerFilter.js]: could you check if this selectable fields from here appear in other places and already put them as separate components with defined values list so they can be reusable, will save work later

File: frontend/src/components/common/headerNavigation.js (hunk 3 above)
Comment (kenzo23312) [reply frontend/src/components/common/headerNavigation.js]: done

File: frontend/src/components/common/headerNavigation.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/components/common/headerNavigation.js]: done

File: frontend/src/components/common/cell/itemRowCellDivider.js (hunk 2 above)
Comment (kenzo23312) [reply frontend/src/components/common/cell/itemRowCellDivider.js]: done

File: frontend/src/components/organizationProfile/hq/hqProfileHeader.js (hunk 5 above)
Comment (kenzo23312) [reply frontend/src/components/organizationProfile/hq/hqProfileHeader.js]: done

File: frontend/src/components/partners/partnerFilter.js (hunk 6 above)
Comment (kenzo23312) [reply frontend/src/components/partners/partnerFilter.js]: done

File: frontend/src/components/common/list/headerList.js (hunk 4 above)
Comment (kenzo23312) [reply frontend/src/components/common/list/headerList.js]: done

**Pull request merged**